src/systems/webxr-dom-overlay.js

```diff
@@ -0,0 +1,21 @@
+var registerSystem = require('../core/system').registerSystem;
+
+/**
+ * WebXR DOM Overlay support.
+ */
+module.exports.System = registerSystem('webxr-dom-overlay', {
+  schema: {
+    root: {default: document.body, type: 'selector'},
+    required: {default: false, type: 'boolean'}
```

Comment:
Maybe calle it `enabled` for consistency with other components?

Reply:
This is something different - if required is true, this is added to WebXR's `requiredFeatures`, so that session creation fails if it's not supported. If false, it's added to `optionalFeatures`, so the session can proceed without a DOM Overlay if that's not supported. It's enabled (if supported) in either case.

Reply:
I've removed this, apps now need to request it as an optional/required feature themselves if they want to use it.